tests/storage/hostdisk.go

```diff
@@ -44,3 +50,19 @@ func CreateDiskOnHost(diskPath string) *k8sv1.Pod {
 
 	return pod
 }
+
+func RemoveHostDiskImage(diskPath string, nodeName string) {
+	virtClient := kubevirt.Client()
+	procPath := filepath.Join("/proc/1/root", diskPath)
+
+	virtHandlerPod, err := libnode.GetVirtHandlerPod(virtClient, nodeName)
+	Expect(err).ToNot(HaveOccurred())
+
+	_, _, err = exec.ExecuteCommandOnPodWithResults(virtHandlerPod, "virt-handler", []string{"rm", "-rf", procPath})
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func RandTmpDir() string {
```

Comment:
This function should not be used for anything but generating paths for hostdisks. Can you find a better name for it (in a followup commit)? e.g `RandHostDiskDir` ?